config/hubs/carbonplan-azure.cluster.yaml

```diff
@@ -0,0 +1,190 @@
+                kubespawner_override:
+                  # Explicitly unset mem_limit, so it overrides the default memory limit we set in
+                  # basehub/values.yaml
+                  mem_limit: null
+                  mem_guarantee: 12G
+                  node_selector:
+                    hub.jupyter.org/node-size: Standard_E2s_v4
+              - display_name: "Medium: E4s v4"
+                description: "~4 CPU, ~30G RAM"
+                kubespawner_override:
+                  mem_limit: null
+                  mem_guarantee: 29G
+                  node_selector:
+                    hub.jupyter.org/node-size: Standard_E4s_v4
+              - display_name: "Large: E8s v4"
+                description: "~8 CPU, ~60G RAM"
+                kubespawner_override:
+                  mem_limit: null
+                  mem_guarantee: 60G
+                  node_selector:
+                    hub.jupyter.org/node-size: Standard_E8s_v4
+              - display_name: "Huge: E32s v4"
+                description: "~32 CPU, ~256G RAM"
+                kubespawner_override:
+                  mem_limit: null
+                  mem_guarantee: 240G
+                  node_selector:
+                    hub.jupyter.org/node-size: Standard_E32s_v4
+              - display_name: "Very Huge: M64s v2"
+                description: "~64 CPU, ~1024G RAM"
+                kubespawner_override:
+                  mem_limit: null
+                  mem_guarantee: 990G
+                  node_selector:
+                    hub.jupyter.org/node-size: Standard_M64s_v2
+              - display_name: "Very Very Huge: M128s v2"
+                description: "~128 CPU, ~2048G RAM"
+                kubespawner_override:
+                  mem_limit: null
+                  mem_guarantee: 2000G
+                  node_selector:
+                    hub.jupyter.org/node-size: Standard_M182s_v2
+            storage:
+              type: none
+              extraVolumes:
+                - name: home
+                  persistentVolumeClaim:
+                    claimName: home-azurefile
+          scheduling:
+            userPlaceholder:
+              enabled: false
+              replicas: 0
+            userScheduler:
+              enabled: false
+          proxy:
+            chp:
+              resources:
+                requests:
+                  cpu: 0.5
+                  memory: 256Mi
+                limits:
+                  cpu: 1
+                  memory: 4Gi
+              nodeSelector: {}
+          hub:
+            resources:
+              requests:
+                cpu: 0.5
+                memory: 256Mi
+              limits:
+                cpu: 1
+                memory: 4Gi
+            allowNamedServers: true
+            readinessProbe:
+              enabled: false
+            nodeSelector: {}
+            config:
+              Authenticator:
+                allowed_users: &users
+                  - <staff_github_ids>
```

Comment:
this should be replaced

Reply:
https://github.com/2i2c-org/infrastructure/blob/2f49365e771b04eeac65a7221172d1d93327ae9b/deployer/utils.py#L51-L69